sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -133,11 +138,27 @@ export class ApplicationStudentsController extends BaseController {
   })
   @ApiBadRequestResponse({ description: "Form validation failed." })
   @ApiNotFoundResponse({ description: "Application not found." })
+  @ApiForbiddenResponse({
+    description: "You have a restriction on your account.",
+  })
   async submitApplication(
     @Body() payload: SaveApplicationDto,
     @Param("applicationId") applicationId: number,
-    @UserToken() userToken: IUserToken,
+    @UserToken() studentToken: StudentUserToken,
   ): Promise<void> {
+    try {
+      await this.applicationControllerService.offeringIntensityRestrictionCheck(
+        studentToken.studentId,
+        payload.data.howWillYouBeAttendingTheProgram,
+      );
+    } catch (error) {
```

Comment:
Can we change this to use the typed version with `catch (error: unknown)`

Reply:
![image](https://user-images.githubusercontent.com/77353155/170738628-9514f95d-bdec-4347-9c35-475862e27bd8.png)
name and message error